network-api/networkapi/wagtailpages/templates/wagtailpages/bannered_campaign_page.html

```diff
@@ -23,3 +23,45 @@
     </div>
   </div>
 {% endblock %}
+
+The following block has been borrowed from the templates/tags/cta/petition.html file,
```

Comment:
Why doesn't this (or the above) need to be in some sort of comment tag?

Reply:
as an `extends` template, anything that isn't inside a block gets entirely ignored. (based on "where would it even render it?" logic)